### PR TITLE
User docsets

### PR DESCRIPTION
**Pull request by kidd**
Ability to download user contributed docsets.  It uses kidd/dashes-to-dashes api as an adapter. In the future, probably official ones will go through there also.

The behaviour of the app should be the same, just call m-x helm-dash-install-user-docset. RET

@areina: we should  test it thoughfully before merge

### CODE REVIEW

**Comment by areina** [comment]
Hey, I tried to install `Arc Lisp` docset and I got this message: `Docset installed. Add "tar: Error is not recoverable: exiting now" to helm-dash-common-docsets or helm-dash-docsets.` `Terraform` docset worked well, so it looks like an issue with that docset. Could we catch the exception?

File: README.md [hunk 1]
```diff
@@ -33,6 +33,14 @@ there is an explanation about stable packages and MELPA and
 
 `m-x package-install helm-dash RET`
 
+
+## Installing docsets
+
+Helm-dash uses the same docsets as [Dash](http://www.kapeli.com/dash).
+You can install them with `m-x helm-dash-install-docset` for the
+official docsets or `m-x helm-dash-install-user-docset` for user
+contributed docsets (experimental).
```
Comment (areina) [inline README.md]: Could you add a reference to new function `helm-dash-install-docset-from-file` ?

**Comment by kidd** [comment]
Hey! Thanks for the review. 

There was an issue with files with spaces. Fixed it :)

**Comment by areina** [comment]
Cool, `Arc Lisp` docset is working fine now. Merging, thanks!

**Pull request merged**